omise/omise-android
# Theming credit card form

### Objective

From previous PRs, we provided basically function for creating the Token via the `CreditCardActivity`. So, this PR continue theming the `CreditCardActivity` to provide the theme and able to custom the theme. 

### Description of changes

On this PR, I renamed the default theme from `OmiseSDKTheme` to better name `OmiseTheme`. Android provided style follow the design.

Custom theme, this PR declared new attributes to able set styles on the `CreditCardActivity`. After the user set styles to attributes on their theme, the layout should apply the set of styles via declared attributes.

```xml
<attr name="omiseButtonStyle" format="reference" />
<attr name="omiseEditTextStyle" format="reference" />
<attr name="omiseEditTextLabelStyle" format="reference" />
<attr name="omiseEditTextErrorStyle" format="reference" />
```


### QA

In this PR has the `SampleTheme` to show how to create a custom theme to use in the user's app. You can change the theme on the `AndroidManifest` to see how it looks on the sample app.

**Example theme**

```xml
 <style name="SampleTheme" parent="Theme.MaterialComponents">
        <item name="android:windowBackground">@color/primary</item>
        <item name="colorPrimary">@color/primary</item>
        <item name="colorPrimaryDark">@color/primary_dark</item>
        <item name="colorAccent">@color/accent</item>
        <item name="omiseEditTextStyle">@style/SampleEditText</item>
        <item name="omiseEditTextLabelStyle">@style/SampleEditTextLabel</item>
        <item name="omiseEditTextErrorStyle">@style/SampleEditTextError</item>
        <item name="omiseButtonStyle">@style/SampleButton</item>
</style>
```

![OmiseTheme](https://user-images.githubusercontent.com/2638321/62672600-9326cc00-b9c5-11e9-8879-ee71047646fb.png)
![SampleTheme](https://user-images.githubusercontent.com/2638321/62672601-93bf6280-b9c5-11e9-899e-8eec6483deee.png)
